ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/sharednetwork/cucushift-installer-rehearse-azure-ipi-sharednetwork-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-sharednetwork
+  steps: 
+    pre:
+    - chain: ipi-conf-azure-sharednetwork
+    - chain: ipi-azure-pre
```

Comment:
`ipi-azure-pre` is something like a wrapper for multiple steps, to easier get understanding which steps are referenced in this workflow, I suggest to flatten the reference steps as much as possible. Something like https://github.com/openshift/release/pull/23891.